rules/azure_signin_rules/azure_legacyauth.yml

```diff
@@ -0,0 +1,298 @@
+AnalysisType: rule
+Filename: azure_legacyauth.py
+RuleID: "Azure.SignIn.LegacyAuth"
+DisplayName: "Azure SignIn via Legacy Authentication Protocol"
+Enabled: true
+Threshold: 1
+DedupPeriodMinutes: 10
+LogTypes:
+  - Azure.SignIn
+Severity: Medium
```

Comment:
this one can totes stay medium but i will suggest low in the event signing in with the legacy auth protocol is not an inherent call to action

Reply:
I expanded the runbook a bit to include some terminology from https://learn.microsoft.com/en-us/azure/active-directory/conditional-access/block-legacy-authentication that explains a bit about why legacy authentication is a security liability. 

I agree that the call to action could be more prescriptive. From the link above ( now also in the runbook ), MSFT makes a strong case for why legacy auth should be totally disabled. I can update the runbook to mention such if you think that makes sense. 

In the current commit, I've included a text blurb from MSFT about the liability of legacy authentication and a link to the msft.com material linked above.